debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -343,7 +343,7 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
                                                  .withDescription("Unique name that identifies the database server and all recorded offsets, and"
                                                          + "that is used as a prefix for all schemas and topics. "
                                                          + "Each distinct MySQL installation should have a separate namespace and monitored by "
-                                                         + "at most one Debezium connector. Defaults to 'host:port'");
+                                                         + "at most one Debezium connector.");
```

Comment:
@rhauch So it actually applies the default as expected if I remove the `isRequired` check. But I'm getting lots of warnings due to the ":" in the name:

    2017-05-18 14:53:02,116 WARN   MySQL|localhost:3306|task  Error while register the MBean 'null': Invalid character ':' in value part of property   [io.debezium.connector.mysql.BinlogReader]
May 18, 2017 2:53:02 PM com.github.shyiko.mysql.binlog.BinaryLogClient connect
WARNING: Binary log position adjusted from 0 to 4
May 18, 2017 2:53:02 PM com.github.shyiko.mysql.binlog.BinaryLogClient connect
INFO: Connected to localhost:3306 at /4 (sid:18780, cid:133)
    2017-05-18 14:53:02,339 WARN   MySQL|localhost:3306|binlog  The Kafka Connect schema name 'localhost:3306.connector_test.products.Value' is not a valid Avro schema name, so replacing with 'localhost_3306.connector_test.products.Value'   [io.debezium.connector.mysql.MySqlSchema]

Should we perhaps use a different concatenation char, so it's e.g. "localhost_3306"? Given that no one can have used the default so far, it should be a safe change. WDTY?